environment

```diff
@@ -148,7 +160,7 @@ DSS_BLOB_PUBLIC_TTL_DAYS=7
 
 # `{account_id}`, if present, will be replaced with the account ID associated with the AWS credentials used for
 # deployment. It can be safely omitted.
-DSS_TERRAFORM_BACKEND_BUCKET_TEMPLATE="org-humancellatlas-dss-{account_id}-${DSS_DEPLOYMENT_STAGE}-terraform"
+DSS_TERRAFORM_BACKEND_BUCKET_TEMPLATE="dss-${DSS_DEPLOYMENT_STAGE}-terraform"
```

Comment:
might want to have a `${platform}` reference here.

Reply:
implemented with 36b3eb0